.github/workflows/ci.yml

```
@@ -17,28 +17,23 @@ name: "CI"
 
 on:
   workflow_call:
-    inputs:
-      jdk:
-        required: false
-        type: string
-        default: '17'
 
 jobs:
   run-unit-tests:
-    name: "unit tests"
+    name: "unit tests(main)"
     strategy:
       fail-fast: false
       matrix:
-        Dtest: [ "A*,F*,S*", "B*,D*,L*,T*", "C*,O*", "E*,N*,Q*", "G*,R*,U*", "H*,I*,J*", "K*,P*,V*,W*,X*,Y*,Z*", "M*"]
+        pattern: [ "L*,Q*,W*", "C*,E*", "B*,O*,S*,X*,Y*,Z*", "A*,G*,R*", "M*,P*,V*", "D*,J*,K*", "F*,H*,U*", "I*,N*,T*"]
```

Nit: Can sort the groups alphabetically, since the order here is how they show up on the UI as well, so just to make it slightly easier for viewers.

```suggestion
        pattern: [ "A*,G*,R*", "B*,O*,S*,X*,Y*,Z*", "C*,E*", "D*,J*,K*", "F*,H*,U*", "I*,N*,T*", "L*,Q*,W*", "M*,P*,V*"]
```